include/truffle.h

```diff
@@ -76,26 +81,24 @@ char truffle_unbox_c(void *object);
 float truffle_unbox_f(void *object);
 double truffle_unbox_d(void *object);
 bool truffle_unbox_b(void *object);
-short truffle_unbox_s(void *object);
 
 // Read
-void *truffle_read(void *object, const char *name);
-int truffle_read_i(void *object, const char *name);
-long truffle_read_l(void *object, const char *name);
-char truffle_read_c(void *object, const char *name);
-float truffle_read_f(void *object, const char *name);
-double truffle_read_d(void *object, const char *name);
-bool truffle_read_b(void *object, const char *name);
-short truffle_read_s(void *object, const char *name);
-
-void *truffle_read_idx(void *object, int idx);
-int truffle_read_idx_i(void *object, int idx);
-long truffle_read_idx_l(void *object, int idx);
-char truffle_read_idx_c(void *object, int idx);
-float truffle_read_idx_f(void *object, int idx);
-double truffle_read_idx_d(void *object, int idx);
-bool truffle_read_idx_b(void *object, int idx);
-short truffle_read_idx_s(void *object, int idx);
+void *	truffle_read 	(void *object, const char *name);
+int 	truffle_read_i 	(void *object, const char *name);
+long 	truffle_read_l 	(void *object, const char *name);
+char 	truffle_read_c 	(void *object, const char *name);
+float 	truffle_read_f 	(void *object, const char *name);
+double 	truffle_read_d 	(void *object, const char *name);
+bool 	truffle_read_b 	(void *object, const char *name);
+
+void *	truffle_read_idx 	(void *object, int idx);
+int 	truffle_read_idx_i 	(void *object, int idx);
+long 	truffle_read_idx_l 	(void *object, int idx);
+char 	truffle_read_idx_c 	(void *object, int idx);
+float 	truffle_read_idx_f 	(void *object, int idx);
+double 	truffle_read_idx_d 	(void *object, int idx);
+bool 	truffle_read_idx_b 	(void *object, int idx);
```

Comment:
Can you also add a short description of the motivation for the changes in the PR?